### ISSUE TITLE
Request filter by regex?

### ISSUE THREAD

**Issue by holloway** [vladlavrik/netify#38]
What options are supported for the request filter and end point?

I want to redirect everything under `/_next/` to a local domain, preferably using [regex backreferences](https://javascript.info/regexp-backreferences) ie,

```
^https:\/\/somesite\.com\/_next\/(.*)$
```
to
```
http://localhost:8000/_next/$1
```

**Comment by vladlavrik** [comment]
I try to keep the Netify interface as simple as possible.
But since there is a request for more complex request transformations like yours, I decided to implement the ability to process the request with code. This will keep the basic configuration simple and at the same time give the opportunity to implement any complexity request mutation.
This feature is under development.

**Comment by vladlavrik** [comment]
The scripting feature was released in version 0.4.0. This feature covers your case.
I close the issue. If you have some difficulties with this, please let me know.
